src/KafkaW/MetaDataQuery.h

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: BSD-2-Clause
+//
+// This code has been produced by the European Spallation Source
+// and its partner institutes under the BSD 2 Clause License.
+//
+// See LICENSE.md at the top level for license information.
+//
+// Screaming Udder!                              https://esss.se
+
+#pragma once
+
+#include <chrono>
+#include <librdkafka/rdkafkacpp.h>
+#include <set>
+#include <string>
+
+namespace KafkaW {
+
+using time_point = std::chrono::system_clock::time_point;
+using duration = std::chrono::system_clock::duration;
+
+// \brief Query the broker for topic + partition offset of a given timestamp.
+//
+// \param[in] Broker    Address of one or several brokers.
+// \param[in] Topic     Name of topic.
+// \param[in] Partition Partition id.
+// \param[in] Time      Timestamp for which the offset should be found.
+// \param[in] TimeOut   The amount of time to wait for the meta-data call to
```

Comment:
Table style alignment of descriptions is nice, but inconsistent with the rest of the code base

Reply:
You can blame clang-format I believe. I do not remember doing it. Feels like a waste of time to change it.

Reply:
Good enough for government work?